uploadNeuroDB/imaging_upload_file_cronjob.pl

```diff
@@ -62,7 +60,7 @@
 &Getopt::Tabular::GetOptions( \@opt_table, \@ARGV ) || exit 1;
 
 { package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
-if ( $profile && !@Settings::db ) {
+if ($profile && !defined @Settings::db) {
```

Comment:
You should remove the defined in front of @Settings::db because of the following warning:
Use of defined is on aggregates (hashes and arrays) is deprecated.